lib/actual_db_schema.rb

```diff
@@ -25,7 +25,8 @@ class << self
 
   self.failed = []
   self.config = {
-    enabled: Rails.env.development?
+    enabled: Rails.env.development?,
+    auto_rollback_disabled: ENV["ACTUAL_DB_SCHEMA_AUTO_ROLLBACK_DISABLED"] == "true"
```

Comment:
```suggestion
    auto_rollback_disabled: !!(ENV["ACTUAL_DB_SCHEMA_AUTO_ROLLBACK_DISABLED"].presence)
```

Let's don't care about the value. If it's something there then it's defined.